content/design-systems-for-developers/react/en/document.md

```diff
@@ -13,23 +13,23 @@ In this chapter, we’ll create a design system “user manual” to help stakeh
 
 ## Documentation is exhausting
 
-It’s obvious – documentation is invaluable for collaborative UI development. It helps teams learn how and when to use common UI components. But why does it take so much effort?
+It’s obvious – documentation is invaluable for collaborative UI development, and it helps teams learn how and when to use common UI components. But why does it take so much effort?
```

Comment:
```suggestion
It’s obvious – documentation is invaluable for collaborative UI development. It helps teams learn how and when to use common UI components. But why does it take so much effort?
```

Revert because the structure is intentional for stylistic reasons.